appinventor/misc/translate/script.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#########################################################
+###     OdeMessages.java converter to .properties     ###
+###   To add compatibility with translation centers   ###
+###          Recommended: OneSky, Transifex           ###
+### ------------------------------------------------- ###
+###     Copyright 2017 - Diego Barreiro, Makeroid     ###
```

Comment:
We would need you to include a statement that this script is released under Apache 2.0 to be able to accept this into App Inventor. Take a look at some of the source files for an example of how to cite the license.